docs/components/choice-tile/index.html

```diff
@@ -107,7 +107,7 @@ <h2 id="basic-choice-tile">Basic Choice Tile</h2>
         </form>
       </header>
 
-      <div class="choice-tile-demo">
```

Comment:
note: class `choice-tile-demo` was removed because wasn't actually doing anything here (used it test page)